packages/@okta/vuepress-site/conductor.yml

```diff
@@ -1499,54 +1467,6 @@ redirects:
     to: /docs/guides/sign-into-spa/vue/use-the-access-token/
   - from: /guides/sign-into-spa/vue/user-info/index.html
     to: /docs/guides/sign-into-spa/vue/user-info/
-  - from: /guides/sign-into-web-app/-/before-you-begin/index.html
```

Comment:
Was this here from before? Maybe we should leave it as-is? We shouldn't be removing previous redirects unless we know for sure that the **from** URL never existed before?

Reply:
I deleted this block of redirects because they were exactly the same as the previous block that I left in.